packages/components/pages/f-contact-preferences/src/components/ContactPreferences.vue

```diff
@@ -177,6 +169,28 @@ export default {
             'editPreference'
         ]),
 
+
+
+        /**
+        * Returns translation string if it exists
+        * @param {string} key - The key of the preference that needs changing
+        */
+        getEmailDescription (key) {
+            if (!this.$te(`${key}.emailDescription`)) return undefined;
```

Comment:
I don't have a big problem with this, but I do think it is generally a little tidier to have a consistent return type 
```suggestion
            if (!this.$te(`${key}.emailDescription`)) return '';
```